CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v3.21.1
+* Support for json.rawMessage when scanning rows
```

Comment:
We use a contract about verbs in CHANGELOG records - always in past feature
`Supported for json.rawMessage when scanning rows`